spec/v2/providers/helpers.ts

```diff
@@ -1,5 +1,20 @@
 import * as options from '../../../src/v2/options';
 
+export const BASIC_OPTIONS: options.EventHandlerOptions = {
```

Comment:
q: Was `FULL_OPTIONS` not appropriate here? IIUC, using `FULL_OPTIONS` wouldn't necessarily add more lines of test code and cover more cases no?

Reply:
You're right, I originally had them in here when I stubbed out `__endpoint` and `FULL_ENDPOINT` didn't exist. Removed them and cleaned up the tests